core/src/test/java/org/apache/calcite/rex/RexExecutorTest.java

```diff
@@ -175,6 +177,58 @@ private void checkConstant(final Object operand,
     });
   }
 
+  @Test public void testUserFromContext() throws Exception {
+    testContextLiteral(SqlStdOperatorTable.USER,
+                       DataContext.Variable.USER, "happyCalciteUser");
+  }
```

Comment:
coding style is to indent arguments 4 spaces, not to line up with '('.

Reply:
done.